openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -636,6 +638,16 @@ def delete_library(library_key):
         raise
 
 
+def _get_library_component_tags_count(library_key) -> dict:
+    """
+    Get the count of tags that are applied to each component in this library, as a dict.
+    """
+    # Create a pattern to match the IDs of the library components, e.g. "lb:org:id*"
+    library_key_pattern = str(library_key).replace("lib:", "lb:", 1) + "*"
+    # TODO: add count_implicit=False when new tagging API version released
```

Comment:
Don't you mean `count_implicit=True` ?
I opened a version bump PR here: https://github.com/openedx/openedx-learning/pull/140

Reply:
Yup, I've update the PR with the new openedx-learning version and added the `count_implicit=True` to it, working as expected now.